src/pages/patientView/PatientViewPage.tsx

```diff
@@ -486,7 +514,24 @@ export default class PatientViewPage extends React.Component<
 
     @action.bound
     onCnaTableRowClick(d: DiscreteCopyNumberData[]) {
-        if (d.length) {
+        // if one cna, select it and toggle off other selected
+        if (d.length == 1) {
+            this.patientViewCnaDataStore.setSelectedCna(d);
```

Comment:
same here. can't we replace this with `this.patientViewCnaDataStore.setSelectedCna([d[0]]);`?